source/getting-started/processing-forms.md

```diff
@@ -178,8 +179,18 @@ public class Register extends ActionSupport {
 }
 ```
 
-In the `Register` class note that we've declared an attribute named `personBean` of type `Person` and there is a public 
-get and set method for this object.
+In the `Register` class, note that we've declared an attribute named `personBean` of type `Person`, there are public 
+getter and setter methods for this object, and the getter is annotated with `@StrutsParameter(depth = 1)`.
+
+In the previous [Coding Struts 2 Actions](coding-actions) tutorial, we annotated the username <strong>setter</strong>,
```

Comment:
`**setter**` instead of `<strong>setter</strong>`?

Reply:
Yeah true, not sure why I wrote this in HTML